ci-scripts/package.sh

```diff
@@ -82,10 +82,10 @@ echo "Building artifacts for "$TARGET
 
 cargo build --release --locked --target $TARGET
 
-if [ "$EXT" = "bin" ]; then
-    echo "Compressing package for $TARGET"
-    tar -cvzf ceramic-one_$TARGET.tar.gz -C $BIN_DIR ceramic-one
-else
+echo "Compressing binary package for $TARGET"
+tar -cvzf ceramic-one_$TARGET.bin.tar.gz -C $BIN_DIR ceramic-one
+
+if [ "$EXT" != "bin" ]; then
```

Comment:
Since we're using a different extension for the zipped binary, should we do the FPM build regardless of `EXT`?

Reply:
And then we can upload both `*.bin.tar.gz` and `*.tar.gz` for all archs.

Reply:
We don't really, I was just trying to simplify the flow to work the same for all cases.